ctru-rs/src/applets/mii_selector.rs

```diff
@@ -270,13 +275,14 @@ impl MiiSelector {
     /// let opts = Options::ENABLE_CANCEL & Options::ENABLE_GUESTS;
     /// mii_selector.set_options(opts);
     ///
-    /// let result = mii_selector.launch()?;
+    /// let result = mii_selector.launch(&apt, &gfx)?;
     /// #
     /// # Ok(())
     /// # }
     /// ```
     #[doc(alias = "miiSelectorLaunch")]
-    pub fn launch(&mut self) -> Result<Selection, Error> {
+    #[allow(unused_variables)]
```

Comment:
nit: Instead of blanket allowing unused variables, rename unused arguments to start with an underscore.